docs/UserGuide.md

```diff
@@ -1034,24 +1102,20 @@ The following features are documented in this section:
     {name: "Storage", url: "#storage"}
 ]%}
 
-#### Saving the data
-
-CCA Manager data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-
-{{ navigateback(navigationlinks) }}
-
----
+<box type="info"><md>
+**Information**: CCA Manager data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+</md></box>
 
 #### Editing the data file
 
-CCA Manager data are saved automatically as a [_JSON_](#glossary) file `[JAR file location]/data/<TODO>.json`. Advanced users are welcome to update data directly by editing that data file.
+CCA Manager data are saved automatically as a [_JSON_](#glossary) file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
To be safe, I think we should rename all `addressbook.json` to `ccamanger.json` too. I'll do that.
```suggestion
CCA Manager data are saved automatically as a [_JSON_](#glossary) file `[JAR file location]/data/ccamanager.json`. Advanced users are welcome to update data directly by editing that data file.
```